src/lib/workspace/AnalysisPanel.tsx

```diff
@@ -107,7 +120,10 @@ export function AnalysisPanel(props: Props) {
           },
           {
             display: 'Visualize',
-            route: `/visualizations${lastVizPath}`,
+            // check whether user is at viz's full screen mode
+            route: inFullscreenVisualization
+              ? '/visualizations/pass-through'
```

Comment:
Should `/pass-through` be hardcoded here?  I don't know how to fix it, but it seems problematic!

Reply:
Actually, the following seems to work just fine (presumably due to default computation redirects)

```
            route: inFullscreenVisualization
              ? '/visualizations'
              : `/visualizations${lastVizPath}`,
```

Reply:
@bobular certainly it can be done programmatically like introducing a function to have string.split('/').pop(), then join('/') but I thought it might be overkill for this case: we can add it later, if necessary

Yeah /visualizations would redirect to /visualizations/pass-through but just left it to be more clear :)

Reply:
@bobular Just for a record, this will work for a single line approach:

`location.pathname.replace(routeBase, '').substring(0, location.pathname.replace(routeBase, '').lastIndexOf('/'))`

Reply:
I'm mostly worried about `pass-through` being hardcoded into the code, when we start using other "computations".

Is your string manipulation suggestion designed to remove the UUID portion of the route, and leave behind the 'pass-through'?  Maybe a regular expression replace (with empty string) would be cleaner (no need to duplicate the `replace(routeBase, '')` )

```
location.pathname.replace(routeBase, '').replace(/\/[^/]*$/, ''); // remove everything from the last '/' to the end
```

Reply:
@bobular yep it works too 👍 I made a new PR using your approach, https://github.com/VEuPathDB/web-eda/pull/193